README.md

```diff
@@ -104,6 +108,47 @@ at runtime as an environment variable:
 
 See `giftless/config.py` for some default configuration options. 
 
+
+### Running a local example
+
+First, create a new project on Github or any other platform.
+Here, we create `example-proj-datahub-io`. Add any data file
+to it. The goal is to track this possible large file with 
+git-lfs and use Giftless as the local server. In our example,
+we create a CSV named `research_data_factors.csv`.
+
+Next, start the Giftless server. Go to your local project 
+directory and run:
+
+```bash
+example-proj-datahub-io$ git init
+example-proj-datahub-io$ git remote add origin YOUR_REMOTE_REPO
+example-proj-datahub-io$ git lfs track '*.csv'
+example-proj-datahub-io$ git lfs track 
+example-proj-datahub-io$ git add .gitattributes #you should have a .gitattributes file at this point
+example-proj-datahub-io$ git add "*.csv"
+example-proj-datahub-io$ git commit -m "Tracking data files"
+example-proj-datahub-io$ git lfs ls-files # you should see your file listed
+example-proj-datahub-io$ git config -f .lfsconfig lfs.url http://127.0.0.1:5000/<user_or_org>/<repo>/
+# in our case, we used http://127.0.0.1:5000/datopian/example-proj-datahub-io/; 
+# make sure to end your lfs.url with /
+example-proj-datahub-io$ git add .lfsconfig
+example-proj-datahub-io$ git commit -m "New git-lfs server endpoint" 
+# if you don't see any changes, run git rm --cached *.csv and then re-add your files, then commit it
+example-proj-datahub-io$ git push origin master
+example-proj-datahub-io$ git lfs push origin master
+```
```

Comment:
To reduce visual complexity it is better to avoid long prompts unless really necessary. 

```suggestion
```bash
$ git init
$ git remote add origin YOUR_REMOTE_REPO
$ git lfs track '*.csv'
$ git lfs track 
$ git add .gitattributes #you should have a .gitattributes file at this point
$ git add "*.csv"
$ git commit -m "Tracking data files"
$ git lfs ls-files # you should see your file listed
$ git config -f .lfsconfig lfs.url http://127.0.0.1:5000/<user_or_org>/<repo>/
# in our case, we used http://127.0.0.1:5000/datopian/example-proj-datahub-io/; 
# make sure to end your lfs.url with /
$ git add .lfsconfig
$ git commit -m "New git-lfs server endpoint" 
# if you don't see any changes, run git rm --cached *.csv and then re-add your files, then commit it
$ git push origin master
$ git lfs push origin master
```
```

Reply:
Big 👍